tools/mat_setup_workspace.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright Google Inc. All Rights Reserved.
+#
+# Use of this source code is governed by an MIT-style license that can be
+# found in the LICENSE file at https://angular.io/license
+
+"Install angular source dependencies"
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
+
+def mat_setup_workspace():
+    """This repository rule should be called from your WORKSPACE file.
+
+    It creates some additional Bazel external repositories that are used internally
+    to build angular material
```

Comment:
Nit: Same with `Angular Material` as above. For sake of consistency and since this is _public API_

Reply:
👍